### PR TITLE
Update the location of the new option

### PR DESCRIPTION
**Pull request by nannanli**
#### Is there a related issue for this PR? 
Issue number: 

#### Developer's Certificate of Origin (DCO)

Signed-off-by: nannanli <nannanli@cn.ibm.com>

### CODE REVIEW

File: docs/user-guide/install-zowe-smpe.md [hunk 1]
```diff
@@ -716,16 +716,62 @@ Uncomment the `VOL=SER=&...` control statements and refer to the comments at the
...
+12. Execute the steps in order. 
+
+    For general information about how to execute z/OSMF workflow steps, watch the [z/OSMF Workflows Tutorial](https://www.youtube.com/watch?v=KLKi7bhKBlE&feature=youtu.be).  
+13. Perform the following steps to execute each step individually:
+
+    1. Double-click the title of the step.
+    2. Select the **Perform** tab. 
+    3. Review the step contents and update the input values as required.
+    4. Select **Next**. 
+    5. Repeat step c and d to complete all items until the opiton **Finish** is avaialable.
```
Comment (sn666239) [inline docs/user-guide/install-zowe-smpe.md]: option and available - it's my spelling mistake.

File: docs/user-guide/install-zowe-smpe.md [hunk 2]
```diff
@@ -716,16 +716,62 @@ Uncomment the `VOL=SER=&...` control statements and refer to the comments at the
...
+13. Perform the following steps to execute each step individually:
+
+    1. Double-click the title of the step.
+    2. Select the **Perform** tab. 
+    3. Review the step contents and update the input values as required.
+    4. Select **Next**. 
+    5. Repeat step c and d to complete all items until the opiton **Finish** is avaialable. 
+    6. Select **Finish**. 
+
+       After you execute each step, the step is marked as **Complete**. The workflow is executed. After you complete executing all the steps individually, the Zowe SMP/E is created.
```
Comment (sn666239) [inline docs/user-guide/install-zowe-smpe.md]: The Zowe SMP/E is created, after you complete the executing all the steps individually. 

Can we move this sentence as conclusion for the z/OSMF procedure..

**Comment by nannanli** [comment]
@BrandonJenkins14 That's a good idea! I've tried to amend my DCO signoff but it does not work. Will try to create a new branch and redo the content. Thanks.

**Comment by nannanli** [comment]
This PR has been replaced by PR #877 . Closing.

**Pull request closed without merging**